wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -97,8 +99,52 @@ Mapsets of songs that originated from other rhythm games often borrow that game'
 - ![](/wiki/shared/diff/normal-s.png) NOVICE / NOV
 - ![](/wiki/shared/diff/hard-s.png) ADVANCED / ADV
 - ![](/wiki/shared/diff/insane-s.png) EXHAUST / EXH
+
+For **SOUND VOLTEX II -infinite infection-** songs:
+
 - ![](/wiki/shared/diff/expert-s.png) INFINITE / INF
 
+For **SOUND VOLTEX III GRAVITY WARS** songs:
+
+- ![](/wiki/shared/diff/expert-s.png) GRAVITY / GRV
+
+For **SOUND VOLTEX IV HEAVENLY HAVEN** songs:
+
+- ![](/wiki/shared/diff/expert-s.png) MAXIMUM / MXM
+- ![](/wiki/shared/diff/expertplus-s.png) HEAVENLY / HVN
+
+For **SOUND VOLTEX VIVID WAVE** songs:
+
+- ![](/wiki/shared/diff/expert-s.png) MAXIMUM / MXM
+- ![](/wiki/shared/diff/expertplus-s.png) VIVID / VVD
+
+If more than provided difficulty names are needed for large mapsets, however, using difficulty names from other versions is allowed.
```

Comment:
this isn't necessary, none of these are hard rules

Reply:
it's still a good practice to reflect the version of the originated game
plus it is actually recommended to do so and new mappers might be unaware of it
thus presenting a reference on the wiki would be a good idea for beginners

Reply:
a recommendation shouldn't be saying "allowed"

Reply:
well i'm not an expert at making up sentences, so suggestions with better phrasing are welcome